.travis.yml

```diff
@@ -24,16 +24,17 @@ jobs:
   include:
     - name: "Rust: stable - Examples"
       rust: stable
-      services:
-        - docker
-      install:
-        - docker pull trufflesuite/ganache-cli:v6.7.0
       script:
-        - docker run -d -p 9545:8545 trufflesuite/ganache-cli:v6.7.0
+        - |
+          mkfifo /tmp/truffle-in
+          tail -f /tmp/truffle-in | yarn --cwd examples/truffle start > /dev/null &
```

Comment:
Why not `tail -f /dev/null` instead of creating your own file?

Reply:
I tried it and it doesn't work. `/dev/null` EOFs right away, and since there is no `STDIN`, tail will exit immediately which will cause the `STDIN` of `truffle develop` to also EOF and then truffle goes down.

Reply:
Didn't know that would work!